src/shogun/classifier/mkl/MKL.h

```diff
@@ -95,6 +95,8 @@ class CMKL : public CSVM
 		 */
 		virtual ~CMKL();
 
+		CSGObject* clone();
```

Comment:
i'm not so sure if i follow... clone is virtual and it needs to stay virtual to override... in c++11 it's override if you like, either way what you do now is not good :) imo